package.json

```diff
@@ -119,9 +119,10 @@
     "@storybook/manager-webpack5": "6.5.15",
     "@storybook/react": "6.5.15",
     "@storybook/theming": "6.5.15",
-    "@swc/cli": "0.1.62",
-    "@swc/core": "1.3.87",
+    "@swc/cli": "0.3.9",
+    "@swc/core": "1.4.2",
     "@swc/helpers": "0.5.1",
+    "@swc/plugin-transform-imports": "1.5.119",
```

Comment:
folks from swc are amazing and this issue have been already fixed in this build https://github.com/swc-project/plugins/issues/246#issuecomment-1959256102

Reply:
Yeah, they are great!